externals/ssl/src/CVMFS-CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+include ("${EXTERNALS_LIB_LOCATION}/CMake-Register_External_Lib.txt")
+
+set (SSL_ARCHIVE "${SSL_BUILTIN_LOCATION}/ssl_install/lib/libssl.a;${SSL_BUILTIN_LOCATION}/ssl_install/lib/libcrypto.a")
```

Comment:
This is not an issue. On macOS, `libcurl` depends on `ssl` and builds it at the beginning of its own configure step, like it does with `c-ares`. Check `externals/libcurl/src/configureHook.sh`. Basically, if the configuration step has completed successfully, `libssl.a` and `libcrypto.a` are already built. This holds also for parallel builds.